README.md

```diff
@@ -1,390 +1,51 @@
-        if (byteArray is byte[]) {
-            string content = check strings:fromBytes(byteArray);
-            log:printInfo("Object content: " + content);
-        }
-    } else {
-        log:printError("Error: " + getObjectResponse.toString());
-    }
-}
-```
-
-### Delete an object
-
-This sample shows how to delete an object that is available in a bucket. The bucket name and object name are required to do this operation. This operation returns an `error` if unsuccessful
-
-```ballerina
-import ballerina/log;
-import ballerinax/aws.s3;
-
-configurable string accessKeyId = ?;
-configurable string secretAccessKey = ?;
-configurable string region = ?;
-configurable string bucketName = ?;
-
-s3:ClientConfiguration amazonS3Config = {
-    accessKeyId: accessKeyId,
-    secretAccessKey: secretAccessKey,
-    region: region
-};
-
-s3:Client amazonS3Client = check new(amazonS3Config);
-
-public function main() {
-    error? deleteObjectResponse = amazonS3Client->deleteObject(bucketName, "test.txt");
-    if (deleteObjectResponse is error) {
-        log:printError("Error: " + deleteObjectResponse.toString());
-    } else {
-        log:printInfo("Successfully deleted object");
-    }
-}
-```
-
-### Delete a bucket
-
-This sample shows how to delete a bucket that is available in an user's account. The bucket name is required to do this operation. This operation returns an `error` if unsuccessful
-
-```ballerina
-import ballerina/log;
-import ballerinax/aws.s3;
-
-configurable string accessKeyId = ?;
-configurable string secretAccessKey = ?;
-configurable string region = ?;
-configurable string bucketName = ?;
-
-s3:ClientConfiguration amazonS3Config = {
-    accessKeyId: accessKeyId,
-    secretAccessKey: secretAccessKey,
-    region: region
-};
-
-s3:Client amazonS3Client = check new (amazonS3Config);
-
-public function main() {
-    error? deleteBucketResponse = amazonS3Client->deleteBucket(bucketName);
-    if (deleteBucketResponse is error) {
-        log:printError("Error: " + deleteBucketResponse.toString());
-    } else {
-        log:printInfo("Successfully deleted bucket");
-    }
-}
-```
-
-
-
-### Pull the Module
-You can pull the Amazon S3 client from Ballerina Central:
-```shell
-$ bal pull ballerinax/aws.s3
-```
+[Amazon Simple Storage Service (Amazon S3)](https://aws.amazon.com/s3/) is an object storage service that offers industry-leading scalability, data availability, security, and performance.This means customers of all sizes and industries can use it to store and protect any amount of data for a range of use cases, such as data lakes, websites, mobile applications, backup and restore, archive, enterprise applications, IoT devices, and big data analytics.
+
+The module provides the capability to manage buckets and objects in AWS S3.
+For more information about configuration and operations, go to the module.
+- [ballerinax/aws.s3](https://docs.central.ballerina.io/ballerinax/aws.s3/1.0.0)
```

Comment:
Dont add version in the link
Please change link to latest agreed format

Reply:
Fixed.